models/NeverLateModel.js

```diff
@@ -29,9 +29,13 @@ const neverLateSchema = new mongoose.Schema(
                     type: String,
                     require: true
                 },
+                Description: {
+                    type: String,
+                    require: false
+                },
                 StartTime: {
                     type: String,
-                    require: true
```

Comment:
why was start time changed to false?? we need it so we can display on the calendar.
please change it back to {require: true}